### PR TITLE
Update kopia commit in kanister-tools image kastenhq 663cbad

### PR DESCRIPTION
**Pull request by leuyentran**
## Change Overview

This PR modifies kopia commit to https://github.com/kastenhq/kopia/commit/663cbad414207f077c666637fabfb22b47855959 in kanister-tools image after Kopia binary upgrade changes.

The commit used here is from forked repo https://github.com/kastenhq/kopia/tree/kopia-at-edfa9ee-minus-32193f7 that is relatively in-sync with upstream at the time of this PR. The same commit is used for go.mod file for kansiter which provides the benefit of synced-context.

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [x] :rainbow: Refactoring (no functional changes, no api changes)
- [x] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [ ] :world_map: Documentation
- [ ] :robot: Test

## Issues 

- fixes #issue-number

## Test Plan

- [x] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -62,9 +62,9 @@ dockers:
   dockerfile: 'docker/tools/Dockerfile'
   build_flag_templates:
   - "--build-arg=kan_tools_version={{ .Tag }}"
-# Refers to https://github.com/kopia/kopia/commit/3551f743d762f2ffe669523272d1c8d734120b79
-  - "--build-arg=kopiaBuildCommit=3551f74"
-  - "--build-arg=kopiaRepoOrg=kopia"
+# Refers to https://github.com/kastenhq/kopia/commit/663cbad414207f077c666637fabfb22b47855959
+  - "--build-arg=kopiaBuildCommit=663cbad"
```
Comment (leuyentran) [inline .goreleaser.yml:66]: @pavannd1 @ankitjain235 Are these `--build-arg` used in `docker/tools/Dockerfile` ?  If so they need to change to `kopia_build_commit` and `kopia_repo_org`

Comment (ankitjain235) [reply .goreleaser.yml:66]: Yes, this seems correct. This seems like a bug from the very beginning. Thanks a lot for noticing it.

Comment (leuyentran) [reply .goreleaser.yml:66]: I'll raise a separate PR for fixing that

**Review by ankitjain235 — COMMENTED** [review]
LGTM.

**Pull request closed without merging**